### PR TITLE
Update Code Insights Direction page with Jason's feedback

### PR DESCRIPTION
**Pull request by Joelkw**
@jyavorska Thanks for the feedback on this page!

I incorporated all your feedback about the page – except the "what's next and why" section. 

We can discuss that feedback in our 1:1 – you noted in your feedback you'd prefer to limit it to the next 3 months, which I agree generally makes sense.

I noted on the page this is for essentially the next six months instead (but really only the next 4.5 months). That would take us through GA launch, as well as Q4 (it felt weird to me to cut Q4, which I'd have to do if I did only want to limit this for the next 3 months since we're still 1 month left in Q3). I did not specify this takes us "through GA" on the page in case we need to move features in/out of GA based on feedback and to limit the number of places we commit to a specific GA date publicly, based on marketing guidance.  

Also, to your questions about whether this is feasible / "seems like a lot" in that timeframe, you are welcome to review our [productboard roadmap](https://sourcegraph.productboard.com/roadmap/3362047-code-insights-beta-to-ga) where we have mapped it out if you're curious.  Some of the bullet points on this page sound large but are truly <1 week of work, if we do them in the right order. As of Monday, the whole team feels that this is an "ambitious but achievable" timeline, and I agree.

### CODE REVIEW

File: content/direction/code-graph/code-insights/index.md [hunk 1]
```diff
@@ -85,8 +85,7 @@ Code Insights is not for trying to track an individual's output and whether or n
 **What this looks like:**
 
 - We don't build features that would primarily be used for evaluating individuals' performance
-- On a case by case basis, we might even intentionally build features to _block_ our other features from being used for this purpose, if the need ever arises
-- We won't market or sell code insights under false pretenses that this is a valid use case for this feature
+- We won't market or sell code insights as a performance management tool
```
Comment (jasonyavorsky) [inline content/direction/code-graph/code-insights/index.md]:
```suggestion
- We won't market or sell code insights as a performance management tool for your team members: it's for looking at the bigger picture, not measuring people
```

**Comment by jasonyavorsky** [comment]
Nice update @Joelkw ! Glad my feedback was helpful. I just expanded on my own recommendation a bit in one suggestion.

> @jyavorska Thanks for the feedback on this page!
> 
> I incorporated all your feedback about the page – except the "what's next and why" section.
> 
> We can discuss that feedback in our 1:1 – you noted in your feedback you'd prefer to limit it to the next 3 months, which I agree generally makes sense.

Sounds good. As with all my feedback I'm not 100% sure if it's a good idea or not, but discussing sounds great.

> I noted on the page this is for essentially the next six months instead (but really only the next 4.5 months). That would take us through GA launch, as well as Q4 (it felt weird to me to cut Q4, which I'd have to do if I did only want to limit this for the next 3 months since we're still 1 month left in Q3). I did not specify this takes us "through GA" on the page in case we need to move features in/out of GA based on feedback and to limit the number of places we commit to a specific GA date publicly, based on marketing guidance.

Perfect, makes total sense.

> Also, to your questions about whether this is feasible / "seems like a lot" in that timeframe, you are welcome to review our [productboard roadmap](https://sourcegraph.productboard.com/roadmap/3362047-code-insights-beta-to-ga) where we have mapped it out if you're curious. Some of the bullet points on this page sound large but are truly <1 week of work, if we do them in the right order. As of Monday, the whole team feels that this is an "ambitious but achievable" timeline, and I agree.

That's awesome. It's really amazing when you find these big impactful things that are also relatively straightforward to implement, or you've structured them in some that you've found a clear value-bringing iteration. That's a sign of great product management!

**Comment by jasonyavorsky** [comment]
@Joelkw just double checking, no further approval is needed from me to merge this.

**Comment by Joelkw** [comment]
(Ah @jyavorska my bad – I think I had to merge back in main because it got out of date but had already checked it off my todo list before I waited for the tests to pass again and re-merge. Will adjust how I am checking my own stale PRs.)

**Pull request merged**